Update Embedded explorer docs with config option changes (#369)
apollographql/docs: src/content/graphos/explorer/sandbox.mdx

<tr>
<td>

##### `hideCookieToggle`

`boolean`

</td>
<td>

> By default, the embedded Sandbox does not show the includeCookies toggle in the connection settings.

Reviewer: ```suggestion
By default, the embedded Sandbox does not show the **Include cookies** toggle in its connection settings.
```

> You can set `hideCookieToggle` to `false` if you want to let users of your embedded Sandbox instance to toggle the `includeCookies` setting in embedded Sandbox.

Reviewer: ```suggestion
Set `hideCookieToggle` to `false` to enable users of your embedded Sandbox instance to toggle the **Include cookes** setting.
```

</td>
</tr>

<tr>
<td>

##### `includeCookies`

`boolean`

</td>
<td>

> By default, the embedded Sandbox uses your `includeCookies` value from your sandbox connection settings.

Reviewer: ```suggestion
By default, the embedded Sandbox uses your `includeCookies` value from your Sandbox connection settings.
```

You can set `includeCookies` to `true` if you instead want the Explorer to pass `{ credentials: 'include' }` for its requests.

If you pass the `handleRequest` option, this option is ignored.

Read more about the `fetch` API and credentials [here](https://developer.mozilla.org/en-US/docs/Web/API/fetch#credentials).

> This config option is **deprecated** in using the sandbox connection settings cookie toggle and setting the default includeCookies value with `initialState.includeCookies`

Reviewer: ```suggestion
This config option is **deprecated** in favor of using the Sandbox connection settings cookie toggle and setting the default value via `initialState.includeCookies`.
```
